index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8">
+    <title>PILAR TORRES</title>
+    <link rel="stylesheet" href="styles/main.css">
+    <link rel="stylesheet" href="about.html">
+    <link rel="stylesheet" href="portfolio.html">
+    <link rel="stylesheet" href="contact.html">
+  </head>
+
+  <body>
+   <header>
+
+      <nav class="navegation-bar">
+        <ul class="navegation-list">
+          <li><a href="about.html">About</a></li>
+          <li><a href="portfolio.html">Portfolio</a></li>
+          <li><a href="contact.html">Contact</a></li>
+        </ul>
+      </nav>
+   </header>
+
+   <main class="main-content">
+      <h1>Pilar TORRES</h1>
+      <h2><button type="button">ENTER</button></h2>
+      <img src="" alt="background">
```

Comment:
If you don't yet have an image, you can use placeimg.com for a placeholder.

Reply:
Perfect! thanks!